### ISSUE TITLE
Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed shouldn't show as warning if leading spaces used in string

### ISSUE THREAD

**Issue by reedy** [squizlabs/PHP_CodeSniffer#722]
When using the Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed checker, it reports problems with leading whitespace being made of spaces when they are in a string. This causes problems for JS and XML (among others) when using large blocks of text

From https://phabricator.wikimedia.org/P2096
```
FILE: ...unit/includes/resourceloader/ResourceLoaderStartUpModuleTest.php
----------------------------------------------------------------------
FOUND 147 ERRORS AFFECTING 147 LINES
----------------------------------------------------------------------
  12 | ERROR | [x] Tabs must be used to indent lines; spaces are not
     |       |     allowed
     |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
  23 | ERROR | [x] Tabs must be used to indent lines; spaces are not
     |       |     allowed
     |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
  26 | ERROR | [x] Tabs must be used to indent lines; spaces are not
     |       |     allowed
```

An example of the code scanned is https://github.com/wikimedia/mediawiki/blob/master/tests/phpunit/includes/resourceloader/ResourceLoaderStartUpModuleTest.php#L21-L30

Please an you fix this? Thanks!

**Comment by reedy** [comment]
Ah. Dupe of https://github.com/squizlabs/PHP_CodeSniffer/issues/515 which is already fixed

Keep up the good work!
